yb-voyager/src/srcdb/source.go

```diff
@@ -86,12 +87,34 @@ func (s *Source) IsOracleCDBSetup() bool {
 
 func (s *Source) ApplyExportSchemaObjectListFilter() {
 	allowedObjects := utils.GetSchemaObjectList(s.DBType)
-	if s.StrExportObjectTypesList == "" {
+
+	if s.StrExportObjectTypesList == "" && s.StrExcludeObjectTypesList == "" {
 		s.ExportObjectTypesList = allowedObjects
 		return
 	}
-	expectedObjectsSlice := strings.Split(s.StrExportObjectTypesList, ",")
+
+	if s.StrExcludeObjectTypesList != "" {
+		excludedObjectsSlice := utils.CsvStringToSlice(s.StrExcludeObjectTypesList)
+		s.ExportObjectTypesList = lo.Filter(allowedObjects, func(objType string, _ int) bool { return !utils.ContainsString(excludedObjectsSlice, objType) })
+		filteredObjects := lo.Filter(excludedObjectsSlice, func(objType string, _ int) bool { return !utils.ContainsString(allowedObjects, objType) })
```

Comment:
Consider using set difference: https://github.com/samber/lo#difference
```
var filteredObjects []string
s.ExportObjectTypeList, filteredObjects = lo.Difference(allowedObjects, excludedObjectSlice)
```

Reply:
You will need to convert all elements of the `excludeObjectSlice` to upper-case.